bbc/simorgh
# FIX: Align FrostedGlassPromo values with Image component props

Resolves JIRA [number]

Overall changes
======
_A very high-level summary of easily-reproducible changes that can be understood by non-devs, and why these changes where made._

Code changes
======

-  Use `srcSet` instead of `srcset` to match the Image component [props](https://github.com/bbc/simorgh/blob/39d9358ee7e38b26f4b42f9a367a2c47ca1d8367/src/app/components/Image/index.tsx#L57), to ensure the `srcset` attribute is included in the rendered `img` element.


Testing
======
1. Visit http://localhost:7080/russian/news-55041160
2. Inspect the promos in the Feature Analysis section.
3. The promos should include the `srcset` attribute in the `img` element.

Before 
![Screenshot 2024-06-19 at 16 16 00](https://github.com/bbc/simorgh/assets/20963836/f0a007a1-b3d2-4798-94e4-d71a94a31ea3)

After
![Screenshot 2024-06-19 at 16 23 26](https://github.com/bbc/simorgh/assets/20963836/0e3c3fba-b403-439e-8d0f-6da1b9dd93a4)


Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
